openlibrary/catalog/add_book/test_add_book.py

```diff
@@ -49,15 +49,28 @@ def ia_writeback(monkeypatch):
     """
     monkeypatch.setattr(add_book, 'update_ia_metadata_for_ol_edition', lambda olid: {})
 
+def test_import_author_personal_name(monkeypatch):
+    """Name order is only flipped to natural order if 'personal_name' is present.
```

Comment:
This feels like it misses some cases, and the significance of `personal_name` isn't clear. Checking whether an author is an individual before flipping the name _is_ a good thing, but this can also be done by checking author `entity_type`, and we should pick a default, is assume personal unless `entity_type` = `org` ?